session_ii/javascript-fundamentals/session1/homework/mpricu/module.js

```diff
@@ -0,0 +1,5 @@
+import { array1 } from "./array.js";
```

Comment:
Doesn't this give you an error? `array1` is exported as a default export.

Reply:
Yes, but in my initial file it was "export {array1}" :((